### PR TITLE
Add "Brass Band" and "Brass Quintet" instrument genres

### PR DESCRIPTION
**Pull request by kmbgit**
Make life easier for finding brass band and brass quintet instruments by adding genres underneath "Brass".
Also create a "Bb Baritone" instrument because that's what brass bands call the "baritone horn" - I should know, I play one!

Resolves: *(direct link to the issue)* There is no issue, just a straight change

I play in a brass band and a brass quintet.  It irks me that there's no easy way to select the instruments that make up these groups. So I created genres for them. Enjoy!

*Use "x" letter to fill the checkboxes below like [x]*

- [x ] I signed [CLA](https://musescore.org/en/cla)
- [ x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [ x] I made sure the code compiles on my machine
- [ x] I made sure there are no unnecessary changes in the code
- [ x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [ x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [ x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made - nope, but is there one?

### CODE REVIEW

**Comment by MarcSabatella** [comment]
These to me don’t make sense as instrumental genres - they are best handled as simple templates.  And indeed, dont these templates already exist?

**Comment by Howard-C** [comment]
I agree, these are not genres, and even if they're added, wood quintet, string quartet/quintet should be added too. But that doesn't make sense either, these are combinations of instruments, not genres.

**Comment by kmbgit** [comment]
Brass Band and Brass Quintet are just as much "genres" as Orchestra, Concert Band, Marching Band, so you can't argue against my addition on that account unless you also argue FOR deleting those other groupings, or having a title of something other than "genre" to group together instruments that typically play together.

If someone wants to add Wood Quintet, String Quartet/Quintet etc then you'll need someone with better knowledge of those genres than me to add them.  But you shouldn't discount something I DO know about on the grounds that I lack the all-round expertise to do the same for other areas.

Templates DO NOT help.  The reason I did this in the first place is that I had a score for a Brass Quintet (Three Brass Cats - https://musescore.com/user/12642511/scores/5858210) and wanted to change Horn in F, Trombone (C, Bass Clef) and Tuba (C, Bass Clef) to Horn in Eb, Trombone (Bb, Treble Clef) and Eb Bass (Treble Clef), and the only way I could find those instruments was to select the "All instruments" group and dig around in that MASSIVE list.

The fundamental problem here is that "genre" is the wrong word to use to categorise things in the "Change Instrument" popup of Stave/Part Properties.  The popup's intent is to reduce "All instruments" into the band subset you're interested in, and that doesn't necessarily follow the musical genre you're writing to.  Can you honestly tell me there's any instrument that you CAN'T play jazz on?

In summary, my PR fulfils the purpose of allowing someone using "Change Instrument" to select from a globally-recognised subset and so find what they're looking for quicker, which is the whole point of subsets.  If you disagree, please remove Orchestra, Concert Band and Marching Band, because I've heard all three of these playing jazz and other musical genres so they're unnecessary duplicates.

**Comment by Howard-C** [comment]
It's indeed inaccurate to say "genre" here, but if we add all these trios/quartets/quintets, the categories list itself can become very crowded.

**Comment by kmbgit** [comment]
Okay, I'll concede on the quintet (quartet, sextet, septet, octet, nonet ...) point as the opportunity for exponential explosion is clear.  In my case it's fair to argue that the "brass quintet" as scored is a subset of the brass parts you would typically find in a Concert (aka Wind) Band under the "brass" subsection, and I wanted to change it to a "brass quintent" formed from a subset of a Brass Band (okay, so I wanted trumpets rather than cornets, but let's not split too many hairs here - that's what the people I play with play on!).  HOWEVER...

I think it's justified to have "Brass Band" as a separate category ("instrument group genre" in MuseScore-speak) because a lot of music is scored for this type of ensemble just as it is for orchestras, concert/wind bands and marching bands, and the Brass Band has some unique instruments including Bb and Eb Bass playing off treble clef and of course the Bb Baritone (my own instrument, if you hadn't already guessed!) whose name is completely wrong in every other context.

So, okay, drop the "quintet", but Brass Band should be allowed.  And definitely add the Bb Baritone!

**Comment by kmbgit** [comment]
I've submitted "Brass Band" sans quintet as https://github.com/musescore/MuseScore/pull/5571

**Comment by Howard-C** [comment]
Then you can close this PR.

**Comment by MarcSabatella** [comment]
For the record, I agree the word “genre” here is not really accurate, and that whatever is really meant by it, it applies more to the whole family of brass band instruments, not to specific groupings like quintets. So I do support the idea behind the modified quintet.

But to clear up some remaining confusion:

It’s not true that you need to hunt around through “All Instruments” to find the brass band instruments - it’s usually much faster to just use the Search box within the window.  And also, assuming you then intend to write additional scores using the same or substantially similar instrumentation, templates definitely *do* help - just save your score to your Templates folder and it’s available as a starting point for future scores.

**Comment by kmbgit** [comment]
Trying again with #5571
